plugin.yaml

```diff
@@ -568,6 +573,13 @@ node_types:
                 delete:
                     implementation: managed_nagios.managed_nagios_plugin.mib.tasks.delete
                     inputs: {}
+    cloudify.nagios.nodes.EnableNagiosRest:
```

Comment:
I'm not sure exactly of the use-case for this, but is a companion DisableNagiosRest desirable?

Reply:
Use case of this is starting nagios in parallel with nodes - this makes sure deployment is faster.